src/mame/casio/rz1.cpp

```diff
@@ -95,16 +93,16 @@ class rz1_state : public driver_device
 	output_finder<> m_led_startstop;
 
 	void map(address_map &map);
+	void pg0_map(address_map& map);
+	void pg1_map(address_map& map);
```

Comment:
Please follow the style of the rest of the file: right-associative qualifiers on the right of the space, i.e.
```
	void pg0_map(address_map &map);
	void pg1_map(address_map &map);
```